Misc/NEWS.d/next/Tests/2023-07-30-18-50-35.gh-issue-107461.i7YjzS.rst

```diff
@@ -0,0 +1 @@
+Test nested ``_as_parameter_`` attribute lookup in :mod:`ctypes`.
```

Comment:
News is not needed for test case changes.

Reply:
Thanks! removed